source/crud/transactions.txt

```diff
@@ -27,28 +27,93 @@ transaction is committed. If any operation in the transaction returns an
 error, the driver cancels the transaction and discards all data changes
 before they ever become visible.
 
+MongoDB guarantees that the data involved in your transaction operations remains
+consistent, even if the operations encounter unexpected errors.
+
+Sessions
+--------
+
 In MongoDB, transactions run within logical **sessions**. A
 :manual:`session </reference/server-sessions/>` is a grouping of related
 read or write operations that you intend to run sequentially. Sessions
-enable :manual:`causal consistency
-</core/read-isolation-consistency-recency/#causal-consistency>` for a
+enable causal consistency for a
 group of operations or allow you to execute operations in an
-:website:`ACID transaction </basics/acid-transactions>`. MongoDB
-guarantees that the data involved in your transaction operations remains
-consistent, even if the operations encounter unexpected errors.
+:website:`ACID transaction </basics/acid-transactions>`.
 
 When using the {+driver-short+}, you can create a new session from a
 ``MongoClient`` instance as an ``IClientSession`` type. We recommend that you reuse
 your client for multiple sessions and transactions instead of
 instantiating a new client each time.
 
+The following example shows how to create a session by calling the ``StartSession()``
+method:
+
+.. code-block::
+
+   var client = new MongoClient("mongodb://localhost:27017");
+   var session = client.StartSession();
+
 .. warning::
 
    Use an ``IClientSession`` only with the ``MongoClient`` (or associated
    ``MongoDatabase`` or ``MongoCollection``) that created it. Using an
    ``IClientSession`` with a different ``MongoClient`` results in operation
    errors.
 
+ClientSessionOptions
+~~~~~~~~~~~~~~~~~~~~
+
+You can customize the behavior of your session by passing an instance of the
+``ClientSessionOptions`` class to the ``StartSession()`` method. The following table
+describes the properties that you can set on a ``ClientSessionOptions`` object:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``CausalConsistency``
+     - Specifies whether the session is causally consistent. In a causally consistent session,
+       the driver executes operations in the order they were issued. To learn more, see
+       :ref:`<csharp-causal-consistency>`.
+       |
+       | **Data Type**: {+bool-data-type+}
+       | **Default**: ``true``
```

Comment:
I: the line breaks aren't working like they should; I think if you add one to the first line of the section, it will fix it:
```suggestion
     - | Specifies whether the session is causally consistent. In a causally consistent session,
          the driver executes operations in the order they were issued. To learn more, see
          :ref:`<csharp-causal-consistency>`.
       |
       | **Data Type**: {+bool-data-type+}
       | **Default**: ``true``
```

Reply:
applies to the other table descriptions

Reply:
whoops--forgot to check staging. thanks